scripts/rest_examples/http-client.env.json

```diff
@@ -6,11 +6,13 @@
     "prefix": "api/provisioning/v1",
     "identity": "eyJpZGVudGl0eSI6IHsidHlwZSI6ICJVc2VyIiwgImFjY291bnRfbnVtYmVyIjoiMTMiLCAiaW50ZXJuYWwiOnsib3JnX2lkIjoiMDAwMDEzIn19fQo=",
     "source_id": "1",
+    "source_id_aws": "1",
+    "source_id_azure": "2",
+    "source_id_gcp": "3",
     "pubkey_id": "1",
     "region": "us-east-1",
     "launch_template_id": "",
     "reservation-get-id": "1",
-    "azure-source-id": "2",
     "azure-resource-group": "redhat-deployed"
   }
 }
```

Comment:
Slight update for consistency so people can easily change their stage source IDs for testing.